lib/generators/geoblacklight/templates/settings.yml

```diff
@@ -229,6 +229,13 @@ WMS_PARAMS:
 # Settings for leaflet
 LEAFLET:
   MAP:
+  BOUNDSOVERLAY:
```

Comment:
WHY ARE WE SHOUTING?

Reply:
Seriously though, how are these values going to get into applications that are being upgraded? This seems like it will only work for newly generated applications.

Reply:
Yeah, that can be an issue, but `settings.yml` changes will go into our v5 upgrade guide. In the past that has sufficed.